src/main/world/glowy-sphere.gd

```diff
@@ -0,0 +1,4 @@
+extends KinematicBody2D
+
```

Comment:
Add commit comment

Reply:
Added comment, `Script for showing a placeholder chattable object in the overworld.`